docs/en/loading/RoutineLoad.md

```diff
@@ -66,7 +66,7 @@ Routine Load now supports consuming CSV, JSON, and Avro (supported since v3.0.1)
 
          > **NOTE**
          >
-         > StarRocks supports access to Kafka via a security authentication mechanism SASL_SSL, SASL or SSL, or without authentication. This topic takes connection to Kafka without authentication as an example. If you need to connect to Kafka via a security authentication mechanism, see [CREATE ROUTINE LOAD](../sql-reference/sql-statements/data-manipulation/CREATE_ROUTINE_LOAD.md).
+         > StarRocks supports access to Kafka via a security protocol SASL_SSL, SAS_PLAINTEXT, SSL, or PLAINTEXT. This topic uses PLAINTEXT to connect to Kafka as an example. If you need to connect to Kafka via other security protocol, see [CREATE ROUTINE LOAD](../sql-reference/sql-statements/data-manipulation/CREATE_ROUTINE_LOAD.md).
```

Comment:
StarRocks supports access to Kafka via security protocols including SASL_SSL, SAS_PLAINTEXT, SSL, and PLAINTEXT. This topic uses connecting to Kafka via PLAINTEXT as an example. If you need to connect to Kafka via other security protocols, xxx